input/pagecontent/resourcereferencesummary.md

```diff
@@ -12,5 +12,6 @@
 |PractitionerRole|[AU Core PractitionerRole](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-practitionerrole.html)|[AU Core Organization](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-organization.html), [AU Core Practitioner](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-practitioner.html)|
 |Procedure|[AU Core Procedure](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-procedure.html)|[AU Core Condition](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-condition.html), [DocumentReference](http://hl7.org/fhir/R4/documentreference.html), [AU Core Encounter](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-encounter.html), [Observation](http://hl7.org/fhir/R4/observation.html), [AU Core Patient](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-patient.html), [AU Core Procedure](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-procedure.html)|
 |RelatedPerson|[AU Base Related Person](https://build.fhir.org/ig/hl7au/au-fhir-base/StructureDefinition-au-relatedperson.html)|-|
-|ServiceRequest|[AU eRequesting Imaging Request](StructureDefinition-au-erequesting-servicerequest-imag.html), [AU eRequesting Pathology Request](StructureDefinition-au-erequesting-servicerequest-path.html)|[AU Base Coverage](https://build.fhir.org/ig/hl7au/au-fhir-base/StructureDefinition-au-coverage.html), [AU Core Encounter](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-encounter.html), [AU Core Patient](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-patient.html), [AU Core PractitionerRole](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-practitionerrole.html)|
+|ServiceRequest|[AU eRequesting Imaging Request](StructureDefinition-au-erequesting-servicerequest-imag.html), [AU eRequesting Pathology Request](StructureDefinition-au-erequesting-servicerequest-path.html)|[AU eRequesting Coverage](StructureDefinition-au-erequesting-coverage.html), [AU Core Encounter](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-encounter.html), [AU Core Patient](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-patient.html), [AU Core PractitionerRole](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-practitionerrole.html)|
```

Comment:
Consider addition of AU eRequesting Diagnostic Request to the ServiceRequest entry. Although it is an abstract profile, it is profiled in eReq.

Reply:
I don't think it should, as the profile is not intended to be instantiated or directly implemented and it is not included in the CapabilityStatement. Listing it may cause confusion. Must Support references from the abstract profile are included in the table through the listed derived profiles.

Ultimately, if we do decide to include it, we should clearly mark it as abstract and explain that it is not directly implemented but is included to provide context for its derived profiles.